### PR TITLE
docs: Rework after super-build docs addition

### PR DESCRIPTION
**Pull request by rleigh-dundee**
Paired with https://github.com/ome/ome-cmake-superbuild/pull/71

- Rename .txt files to .rst
- Delete content moved to the super-build
- Pick up some changes from the super-build
- Update the schema docs for 2016-06

### CODE REVIEW

File: docs/sphinx/themes/globalbftoc.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<h3><a href="{{ pathto(master_doc) }}">{{ _('OME-Files') }}</a></h3>
+<h3><a href="{{ pathto(master_doc) }}">{{ _('Bio-Formats') }}</a></h3>
 {{ toctree()}}
-<a href="http://downloads.openmicroscopy.org/latest/bio-formats5.1/">{{ _('Main Downloads') }}</a></br>
-<a href="http://downloads.openmicroscopy.org/latest/bio-formats-cpp5.1/">{{ _('C++ Downloads') }}</a></br>
-<a href="//www.openmicroscopy.org/site/about/licensing-attribution">{{ _('Licensing') }}</a>
+<a href="http://downloads.openmicroscopy.org/latest/bio-formats5.2/">{{ _('Bio-Formats Downloads') }}</a></br>
```
Comment (sbesson) [inline docs/sphinx/themes/globalbftoc.html]: Do we need a reference to the Bio-Formats downloads at all here ?

File: docs/sphinx/themes/globalbftoc.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<h3><a href="{{ pathto(master_doc) }}">{{ _('OME-Files') }}</a></h3>
+<h3><a href="{{ pathto(master_doc) }}">{{ _('Bio-Formats') }}</a></h3>
```
Comment (sbesson) [inline docs/sphinx/themes/globalbftoc.html]: 'OME Files' ?

**Comment by sbesson** [comment]
As briefly mentioned elsewhere, happy to consider the `txt -> rst` policy change but PRs should be simultaneously opened across all relevant repositories containing Sphinx documentation to assess the impact of the switch at the level of the organization.

File: docs/sphinx/overview.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
+OME Files C++ overview
+======================
+
+OME Files C++ is a reference implementation of the OME data model and
+the OME-TIFF file format for C++ developers who wish to integrate
+support for the OME data model and reading and writing the standard
+OME-TIFF file format into their software.  Potential uses include
+export of images using OME-TIFF, saving of acquired image data in
+OME-TIFF, reading metadata and image data from OME-TIFF for
+visualisation and analysis, or use of the data model metadata APIs for
```
Comment (hflynn) [inline docs/sphinx/overview.rst]: Before Mark spots it - visualization

File: docs/sphinx/overview.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+handling metadata.
+
+OME Files C++ is a re-implementation of the Bio-Formats Java API,
+providing equivalent data model, metadata and reading and writing
+interfaces.  Unlike the Bio-Formats Java API, which supports reading
+of over 130 file formats and writing of over 15 file formats at the
```
Comment (hflynn) [inline docs/sphinx/overview.rst]: over 140 file formats

File: docs/sphinx/overview.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+time of writing, OME Files C++ restricts itself to reading and writing
+OME-TIFF and plain TIFF, and does not at present support additional
+formats.  At present it is a reference implementation for the OME-TIFF
```
Comment (hflynn) [inline docs/sphinx/overview.rst]: You need to get rid of one of the 'at present's here - these two sentences are a bit repetitive

File: docs/sphinx/overview.rst [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+format, and the focus for development is upon improving and extending
+the data model metadata APIs and the reader and writer APIs, rather
+than adding additional formats.
+
+.. note::
+  Due to the renaming of Bio-Formats C++ to OME Files, this will
```
Comment (hflynn) [inline docs/sphinx/overview.rst]: 👍 to the clarification that only BF C++ has been renamed

File: docs/sphinx/overview.rst [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  result in an API break between version Bio-Formats C++ 5.1 and OME
+  Files C++ 0.x as a result of the renamed namespaces.  Further
```
Comment (hflynn) [inline docs/sphinx/overview.rst]: "...due to the renamed.."? Two 'result's in this sentence is repetitive

File: docs/sphinx/themes/globalbftoc.html (hunk 1 above)
Comment (hflynn) [inline docs/sphinx/themes/globalbftoc.html]: I would think not

**Comment by sbesson** [comment]
See https://github.com/ome/ome-cmake-superbuild/pull/71#issuecomment-22995105. Merging. Can the comments above be addressed in a follow-up PR ?

**Pull request merged**